tox.ini

```diff
@@ -28,6 +22,19 @@ commands_post =
     # This is to deal with running the containers with --detached
     docker-compose down
 
+[testenv:linters]
+commands_pre =
+    pip install -r {toxinidir}/requirements/dev.txt
+    pip install -r {toxinidir}/requirements/common.txt
+commands =
+    pip check
+    {toxinidir}/lints/queuelint.py
+    flake8 --show-source
+    black --check .
+    shellcheck initialize_data.sh
+    shellcheck docker/entrypoint.sh
+commands_post =
```

Comment:
Great - It makes more sense to break these python lint checks into a separate command like this.